lib/Doctrine/Migrations/Metadata/AvailableMigrationsSet.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Migrations\Metadata;
+
+use Countable;
+use Doctrine\Migrations\Exception\MigrationNotAvailable;
+use Doctrine\Migrations\Version\Version;
+use function array_values;
+use function count;
+
+final class AvailableMigrationsSet implements Countable
+{
+    /** @var AvailableMigration[] */
+    protected $items = [];
```

Comment:
I would make it `private` (as the class is final, protected does not have any benefit anyway)